src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,22 @@
 import React from 'react';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
-);
+import './UserInfo.scss';
+
+import { User } from '../../types/User';
+
+type Props = {
+  user: User
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const {
+    name,
+    email,
+  } = user;
```

Comment:
```suggestion
  const { name, email } = user;
```
You can leave it in one line if amount of props < 3